demo/index.js

```diff
@@ -165,6 +172,12 @@ class App extends React.Component {
             </Tippy>
           </Tippy>
         </Tippy>
+
+        <h1>Other</h1>
+        <input type="text" placeholder="Enter class" onChange={this.updateCustomClass}/>
+        <Tippy placement="bottom" className={this.state.customClass}>
+          <button>Custom class</button>
+        </Tippy>
```

Comment:
Not the prettiest test, but it helped to confirm the classes were being dynamically added/removed from the tippy element.